src/Livewire/DataTables/ContactList.php

```diff
@@ -37,13 +42,19 @@ class ContactList extends DataTable
         'avatar' => 'image',
     ];
 
+    public Contact $contact;
+
     public function getTableActions(): array
     {
         return [
             DataTableButton::make()
                 ->label(__('Create'))
                 ->color('primary')
-                ->icon('plus'),
+                ->icon('plus')
+                ->attributes([
+                    'x-on:click' => '$wire.show()',
+                ])
+                ->when(fn () => auth()->user()->can('create', $this->model)),
```

Comment:
this works?
Also this->model = Address, but here the user can create a contact. Address != Contact